### PR TITLE
[Rule Tuning] Persistence via Update Orchestrator Service Hijack

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/937

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by brokensound77 — APPROVED** [review]
thanks, LGTM 👍

File: rules/windows/persistence_via_update_orchestrator_service_hijack.toml [hunk 1]
```diff
@@ -11,21 +11,29 @@ level of SYSTEM.
...
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.parent.name:svchost.exe and
-  process.parent.args:(UsoSvc or usosvc) and
-  not process.name:(UsoClient.exe or usoclient.exe or MusNotification.exe or musnotification.exe or MusNotificationUx.exe or musnotificationux.exe)
+process where event.type : "start" and
```
Comment (bm11100) [inline rules/windows/persistence_via_update_orchestrator_service_hijack.toml:25]:
```suggestion
process where event.type == "start" and
```

**Review by bm11100 — APPROVED** [review]
🚢

**Comment by willemdh** [comment]
@Samirbous @bm11100 
process.name WerFault.exe should also be excluded.

**Comment by Samirbous** [comment]
@willemdh its already [excluded](https://github.com/elastic/detection-rules/pull/939/files#diff-4b57940bbda4fb48cf0d486ab679253682ded8d458fffba9fbf63b14d639986fR34).

**Pull request merged**